### PR TITLE
Update unratedTaskTable styles to match Figma design

### PR DESCRIPTION
**Pull request by PatricKangasmaki**

### CODE REVIEW

File: frontend/src/components/TableUnratedTaskRow.tsx [hunk 1]
```diff
@@ -113,60 +114,59 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
...
+        modalType: ModalTypes.RATE_TASK_MODAL,
+        modalProps: {
+          taskId: task.id,
+        },
+      }),
+    );
   };
 
-  const openModal = (modalType: ModalTypes) => (e: React.SyntheticEvent) => {
+  const taskDetailsClicked = (e: React.MouseEvent<any, MouseEvent>) => {
```
Comment (IidaKainu) [inline frontend/src/components/TableUnratedTaskRow.tsx:130]: Both taskDetailsClicked and rateTaskClicked could use openModal
```
const openModal = (modalType: ModalTypes) => (e: React.SyntheticEvent) => {
    e.preventDefault();
    e.stopPropagation();
    dispatch(
      modalsActions.showModal({
        modalType,
        modalProps: {
          taskId: task.id,
        },
      }),
    );
  };
```
The modal type would be sent on event handlers
`onClick={openModal(ModalTypes.TASK_INFO_MODAL)}`
`onClick={openModal(ModalTypes.RATE_TASK_MODAL)}`

File: frontend/src/components/TableUnratedTaskRow.tsx [hunk 2]
```diff
@@ -227,12 +227,18 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
...
+            type="button"
+            onClick={notifyUsersClicked}
+          >
+            <Trans i18nKey="Notify" />
+          </button>
+        )}
+        {taskAwaitsRatings(task, userInfo) ? (
           <a
             href={modalLink(ModalTypes.TASK_RATINGS_INFO_MODAL, {
               taskId: task.id,
```
Comment (IidaKainu) [inline frontend/src/components/TableUnratedTaskRow.tsx:244]: I am not sure if this link is needed (the correct url is shown without it too), but shouldn't the ModalTypes be RATE_TASK_MODAL?

**Review by IidaKainu — CHANGES_REQUESTED** [review]
The unratedTaskTable looks just like Figma's 👍 
Just a few things I noticed that could be cleared up

**Comment by PatricKangasmaki** [comment]
I added all your suggestions and tested the functionalities again (everything works as wanted) 👍

**Review by IidaKainu — APPROVED** [review]

**Pull request merged**